client/src/components/Donation/DonateForm.js

```diff
@@ -335,9 +428,45 @@ class DonateForm extends Component {
       </Row>
     );
   }
+
+  render() {
+    const {
+      donationFormState: { processing, success, error },
+      isMinimalForm
+    } = this.props;
+    if (success || error) {
```

Comment:
```suggestion
    if (success || error || processing) {
```
Can't we simplify things by including processing here?

Reply:
That would remove the payment provider element from the DOM and payment will be interrupted. 
Therefore I still keep the elements in the DOM but hide them and show the processing message.

Reply:
Understood, thanks for explaining.